netlify/pages/about.yaml

```
@@ -5,6 +5,31 @@ hero:
     alt: About Us
     src: /images/team/team.jpg
   title: About Stoplight
+valueSection:
+  sectionTitle: Our Values
```

I'm indifferent on this change. Technically these aren't _all_ of our values, just our "core" ones

```suggestion
  sectionTitle: Our Core Values
```